tests/test_inference.py

```diff
@@ -14,6 +14,7 @@ def test_one_parameter_interval_nonlincontr(xes):
 
     lf = fd.LogLikelihood(
         sources=dict(er=xes.__class__),
+        arguments=dict(er=dict()),
```

Comment:
Can you init arguments to an empty dict by default? Then users won't have to add these dummy dicts all the time.

Reply:
Done.